docs/kind.md

```diff
@@ -132,8 +128,6 @@ These instructions assume that you have built the Antrea Docker image locally
 (e.g. by running `make` from the root of the repository).
 
 ```bash
-# "fix" the host's veth interfaces (for the different Kind Nodes)
-kind get nodes | xargs ./hack/kind-fix-networking.sh
```

Comment:
better to remove `--kind` below or change to `kubectl apply -f build/yamls/antrea.yml`?